src/core/server/http/ssl_config.ts

```diff
@@ -49,6 +49,7 @@ export const sslSchema = schema.object(
       schema.oneOf([schema.literal('TLSv1'), schema.literal('TLSv1.1'), schema.literal('TLSv1.2')]),
       { defaultValue: ['TLSv1.1', 'TLSv1.2'], minSize: 1 }
     ),
+    requestCert: schema.maybe(schema.boolean({ defaultValue: false })),
```

Comment:
@toddself Hmm, IIRC you either use `schema.maybe(schema.boolean()) // boolean?` or just `schema.boolean({ defaultValue: false }) // boolean` as having default value implies "optionality". Or I'm missing something?